src/cache/cache_las.c

```diff
@@ -1336,7 +1348,8 @@ __wt_find_lookaside_upd(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, WT_UPDAT
      * timestamp is part of the key, our cursor needs to go from the newest record (further in the
      * las) to the oldest (earlier in the las) for a given key.
      */
-    for (ret = __wt_las_cursor_position(session, cursor, las_btree_id, key, txn->read_timestamp);
+    for (ret = __wt_las_cursor_position(session, cursor, las_btree_id, key,
+           allow_prepare ? txn->prepare_timestamp : txn->read_timestamp);
```

Comment:
This is harder to read and doesn't gain us space, I'd pull the function call out:
```
ret = __wt_las_cursor_position(session, cursor, las_btree_id, key,
  allow_prepare ? txn->prepare_timestamp : txn->read_timestamp;
for (;  ret == 0; ret = cursor->prev(cursor)) {
```

Reply:
721875b